src/components/LabeledRadioButton/LabeledRadioButton.less

```diff
@@ -0,0 +1,41 @@
+.lucid-LabeledRadioButton {
+	align-items: center;
+	color: @color-darkGray;
+	cursor: pointer;
+	display: flex;
+	font: @font;
+	justify-content: flex-start;
+
+	&&-is-disabled {
+		color: @color-mediumGray;
+		cursor: not-allowed;
+	}
+
+	&:hover .lucid-RadioButton-visualization-container {
+		border-color: @color-primary;
+		transition: @RadioButton-animation-timing;
+	}
+
+	&:hover .lucid-RadioButton-visualization-glow {
+		opacity: 0.3;
```

Comment:
Didn't realize there was one. Nice. I don't think _any_ of our components are using that mixin right now...